backend/tests/conftest.py

```
@@ -22,7 +22,7 @@
 from tarsy.utils.timestamp import now_us
 
 # Import all database models to ensure they're registered with SQLModel.metadata
-from tarsy.models.history import AlertSession, StageExecution
+from tarsy.models.db_models import AlertSession, StageExecution
```

Ruff (0.12.2)

`tarsy.models.db_models.AlertSession` imported but unused

Remove unused import

(F401)